src/components/templates/Search/index.module.css

```
@@ -2,6 +2,34 @@
   display: grid;
 }
 
+row {
```

missing a dot so these styles were not kicking in:

```suggestion
.row {
```